pkg/controller/backup_deletion_controller.go

```diff
@@ -289,6 +294,31 @@ func (c *backupDeletionController) processRequest(req *velerov1api.DeleteBackupR
 		errs = append(errs, err.Error())
 	}
 
+	// Download the tarball
+	backupFile, err := downloadToTempFile(backup.Name, backupStore, log)
+	if err != nil {
+		return errors.Wrap(err, "error downloading backup")
+	}
+	defer closeAndRemoveFile(backupFile, c.logger)
+
+	actions, err := pluginManager.GetDeleteItemActions()
+	if err != nil {
+		return errors.Wrap(err, "error getting delete item actions")
+	}
+	// don't defer CleanupClients here, since it was already called above.
+
+	ctx := &delete.Context{
+		BackupReader:    backupFile,
+		Actions:         actions,
+		Log:             c.logger,
+		DiscoveryHelper: c.helper,
+		Filesystem:      nil,
```

Comment:
should this be nil? or is this a `TODO`?

Reply:
Actually I think it should be populated; I populated it elsewhere in the code right now, but I'll fix that up.